data/windows.yaml

```diff
@@ -320,6 +320,30 @@ urls:
 - 'https://www.fireeye.com/content/dam/fireeye-www/global/en/current-threats/pdfs/wp-windows-management-instrumentation.pdf'
 - 'https://github.com/libyal/dtformats/blob/main/documentation/WMI%20repository%20file%20format.asciidoc'
 ---
+name: WindowsCIMDatabase
```

Comment:
There is already WindowsCIMRepositoryFiles, btw this definition is also not correct. The CIM repository consists of more than OBJECTS.DATA

Reply:
I realise that it's missing the other files, but that was the intent of this definition. I only wanted to grab OBJECTS.DATA, which contains the data itself (without the definitions on how to parse it, etc.). AFAIK, if you run scripts that carve out various data structures as linked in the 'urls' section, then this file is all you need, and collecting others is unnecessary (and reduces the size of what we collect). WDYT?

Reply:
I would just collect all the files, since there are many caveats to reconstructing data from OBJECTS.DATA files

Reply:
As per our side chat, I'll remove this definition and thanks for your work on https://github.com/libyal/dtformats/blob/main/scripts/wmi_repository.py 🥇

Reply:
I've updated the URLs in the definition and added the paths of the "copies created during system rebuild".